flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/ResultPartition.java

```diff
@@ -466,4 +465,35 @@ private void notifyPipelinedConsumers() {
 			hasNotifiedPipelinedConsumers = true;
 		}
 	}
+
+	private static void initializeBoundedBlockingPartitions(
```

Comment:
I guess this method would have other usages such as tests in future? Otherwise we do not need to pass these parameters here.

Reply:
yes